### ISSUE TITLE
VPX.framework and armv7

### ISSUE THREAD

**Issue by kobelb** [brion/OGVKit#129]
When trying to build the Example application for armv7, I'm getting the following errors:

```
Undefined symbols for architecture armv7:
  "_vp8_sixtap_predict4x4_armv6", referenced from:
      _vp8_decode_frame in VPX(decodeframe.c.o)
  "_vp8_bilinear_predict4x4_armv6", referenced from:
      _vp8_decode_frame in VPX(decodeframe.c.o)
  "_vp8_intra4x4_predict_armv6", referenced from:
      _mt_decode_mb_rows in VPX(threading.c.o)
      _vp8_decode_frame in VPX(decodeframe.c.o)
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I was able to update the libvpx framework to one that I compiled locally, and then it works correctly. I noticed that you specified a specific snapshot for libvpx and was wondering whether that's necessary still

**Comment by brion** [comment]
Darn, I thought I'd fixed that... Lemme check the specs repo.

**Comment by brion** [comment]
(Note the root problem is with building VPX under Xcode 7; looks like I left a broken version in the repo.)

**Comment by kobelb** [comment]
Gotcha. I think someone fixed that and it was merged into the master branch a few days ago.

**Comment by brion** [comment]
Ok, looks like latest versions now build under Xcode 7 but still have the bug with armv7; I'm re-downloading xcode 6.4 to make a fresh build under the older SDK.

**Comment by brion** [comment]
Ok should be fixed as of f233959 (you might have to remove the Pods dir & Podfile.lock and re-run 'pod install')

**Comment by kobelb** [comment]
Awesome! Thanks for the awesome library and doing all this.
